packages/design-system/src/components/Banner/Banner.test.js

```diff
@@ -0,0 +1,32 @@
+import * as React from 'react';
+import { shallow } from 'enzyme';
+import CloseIcon from 'react-material-icon-svg/dist/CloseIcon';
+
+import Banner from './Banner';
+
+describe('Banner', () => {
+  it('should render small info Banner', () => {
+    const sizes = ['small', 'large', 'medium'];
+    const types = ['info', 'warning', 'success', 'error'];
+    const onClose = jest.fn();
+
+    types.map(type => {
+      sizes.map(size => {
+        const component = shallow(<Banner text="Example text" type={type} size={size} onClose={onClose} />);
+
+        expect(component).toMatchSnapshot();
+      })
+    })
+  });
+
+  it('should call on close on close button click', () => {
+    const onClose = jest.fn();
+    const component = shallow(<Banner text="Example text" onClose={onClose} />);
+    component.find(CloseIcon).simulate('click')
+    expect(onClose.mock.calls.length).toEqual(1);
```

Comment:
`expect(onClose).toHaveBeenCalled()`

Reply:
eventually `toHaveBeenCalledTimes(1)` if the number of calls is the thing here :-)